### ISSUE TITLE
Allow manual triggering of removing old logs

### ISSUE THREAD

**Issue by JWardee** [JWardee/wp-mail-catcher#136]
Add a button to the settings page that will allow the cron job to be manually triggered. This should help out https://wordpress.org/support/topic/auto-delete-still-not-working/

**Comment by Nemi5150** [comment]
Yes, the problem is that `posts_per_page `defaults to 5, so it **is** deleting 5 emails per cron job right now. But passing in `posts_per_page = -1` results in it deleting all expired emails.

```
foreach (Logs::get(array('posts_per_page' => -1)) as $log) {
            error_log(print_r($log['timestamp'], true));
            if ((time() - $log['timestamp']) >= $interval) {
                $idsToRemove[] = $log['id'];
            }
        }
```

**Comment by JWardee** [comment]
Solved in 0658780eb58b00c8092fecb495c67e4f0d4ed311
